content/quickstart/kubernetes/tutorial-workloads.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Tutorials - Workloads
+aliases: ["/reference/kubernetes-workloads.html"]
```

Comment:
```suggestion
```

I don't think this alias (redirect) is needed. Did we previously have a `/reference/kubernetes-workloads.html` file?